AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -80,25 +89,35 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId {
 
 - (BOOL)loadLogsWithGroupId:(NSString *)groupId
                       limit:(NSUInteger)limit
-             withCompletion:(nullable MSLoadDataCompletionBlock)completion {
+                      iKeys:(nullable NSArray< NSString *> *)iKeys
+          completionHandler:(nullable MSLoadDataCompletionBlock)completionHandler {
```

Comment:
It probably makes sense to rename `MSLoadDataCompletionBlock` to `MSLoadDataCompletionHandler` now that `completion` is renamed `completionHandler`.